source/renderer/app/utils/walletUtils.js

```diff
@@ -1 +1,13 @@
+// @flow
+import BigNumber from 'bignumber.js';
+
+// $FlowFixMe TODO: fix this in flowconfig
 export default import('@iohk-jormungandr/wallet-js').then((modules) => modules);
+
+const MINIMUM_BALANCE_FOR_REWARD: number = 10;
+
+export const isWalletEmptyWitoutRewards = (
+  totalAmount: BigNumber,
+  available: BigNumber
```

Comment:
Please rename to `availableAmount` or `walletBalance` (whichever you think is better)